Camera.js

```diff
@@ -186,6 +186,22 @@ export default class Camera extends Component {
     }
   }
 
+  releaseCamera(options) {
+    const props = convertStringProps(this.props);
+    options = {
+      audio: props.captureAudio,
+      mode: props.captureMode,
+      playSoundOnCapture: props.playSoundOnCapture,
+      target: props.captureTarget,
+      quality: props.captureQuality,
+      type: props.type,
+      title: '',
+      description: '',
+      ...options
+    };
+    CameraManager.releaseCamera(options);
+  }
+
```

Comment:
The releaseCamera method expects "type" to be in the options array, you pass in a lot of other stuff which seems unnecessary?